tests/AlertTest.php

```diff
@@ -0,0 +1,137 @@
+<?php
+
+namespace SynergiTechAlert\Tests;
+
+use Orchestra\Testbench\TestCase as BaseTestCase;
+use SynergiTech\Alert\ServiceProvider;
+use SynergiTech\Alert\Facade;
+use BadMethodCallException;
+use Alert;
+
+class AlertTest extends BaseTestCase
+{
+    /**
+     * Setup testbench
+     */
+    protected function getPackageProviders($app)
+    {
+        return [ServiceProvider::class];
+    }
+
+    protected function getPackageAliases($app)
+    {
+        return [
+            'Alert' => Facade::class,
+        ];
+    }
+
+    /**
+     * cheeky helper
+     */
+
+    private function doTheAssertions(array $input)
```

Comment:
Can we change this to be like `assert*` and a bit more descriptive, maybe something like `assertSessionAlertEquals(array $expectedAlert)`?